tools/lkl/lib/net.c

```diff
@@ -3,6 +3,67 @@
 #include "endian.h"
 #include <lkl_host.h>
 
+#ifdef __MINGW32__
+#include <ws2tcpip.h>
+#endif
+
+#ifdef __MINGW32__
+/**
+ * code is referred from
+ * https://rt.cpan.org/Public/Bug/Display.html?id=91524
+ */
```

Comment:
not sure about the license, is it GPLv2 compatible?

Reply:
good catch.
I revised this part in a different way.